tests/utils/parallel.py

```diff
@@ -43,7 +43,7 @@ def test_run_in_parallel_batch_size():
     result_parallel = run_in_parallel(
         func, data, n_jobs=-1, batch_size=1, flatten_results=True
     )
-    assert result_sequential == result_parallel
+    assert result_sequential == result_parallel 
 
 
```

Comment:
And here we would need more tests. Probably `leave and maybe `dynamic_ncols` should be tested, by using `capsys`. You can either test it on some simple functions like in `test_progress_parallel` (more like unit tests), or compute fingerprints (more integration test style).